tests/bluetooth/tester/src/btp/btp_ascs.h

```diff
@@ -0,0 +1,85 @@
+/* btp_bap.h - Bluetooth tester headers */
+
+/*
+ * Copyright (c) 2023 Codecoup
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+/* ASCS commands */
+#define BTP_ASCS_READ_SUPPORTED_COMMANDS	0x01
+struct btp_ascs_read_supported_commands_rp {
+	uint8_t data[0];
+} __packed;
+
+#define BTP_ASCS_CONFIGURE_CODEC	0x02
+struct btp_ascs_configure_codec_cmd {
+	bt_addr_le_t address;
+	uint8_t ase_id;
+	uint8_t coding_format;
+	uint8_t freq;
+	uint8_t frame_duration;
+	uint32_t audio_locations;
+	uint16_t octets_per_frame;
+} __packed;
```

Comment:
Is the BTP definitions somewhere public where I can compare these to the BTP spec?

Reply:
merged specs are here  https://github.com/auto-pts/auto-pts/tree/master/doc

while that service is stil in WIP  https://github.com/auto-pts/auto-pts/pull/866 (I already asked Magda to add doc update in that PR)